### PR TITLE
[WIP] Continuous PR to monitor changes in develop branch

### PR DESCRIPTION
**Pull request by python4eg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "rv-026webui",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "devserver": "node ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --debug --hot --devtool eval-source-map --output-pathinfo --watch --colors --inline --content-base ./ --port 8090 --host 0.0.0.0",
```
Comment (python4eg) [inline package.json]: migrate to gulp

File: pg_config.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+var pasword = "postgres";
+var dbServerLocation = "localhost";
+var serverPort = "5432";
+var dbName = "testdb";
+
+var str = `postgres://${userName}:${pasword}@${dbServerLocation}:${serverPort}/${dbName}`;
+
+module.exports = {
+    str
+  };
```
Comment (python4eg) [inline pg_config.js]: newline

File: static/js/app/store.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import {createStore, applyMiddleware, compose} from 'redux';
+import {browserHistory} from 'react-router';
+import {routerMiddleware} from 'react-router-redux';
+import {createLogger} from 'redux-logger';
+import rootReducer from './reducer';
+
+
+const middlewares = [routerMiddleware(browserHistory)];
+
+if(process.env.NODE_ENV !== 'production') {
```
Comment (python4eg) [inline static/js/app/store.js]: create a separate file with constants to hold all environments

File: package.json (hunk 1 above)
Comment (l1s4un) [reply package.json]: @python4eg  Migration to gulp [done](https://github.com/Social-projects-Rivne/Rv-026WebUI/pull/18)!

File: static/js/app/store.js (hunk 3 above)
Comment (iuriifaevskii) [reply static/js/app/store.js]: @python4eg separate file with environments constants was [created](https://github.com/Social-projects-Rivne/Rv-026WebUI/blob/b99e987113669797db23af493b2faa4f793afbb7/config.js)!

File: pg_config.js (hunk 2 above)
Comment (l1s4un) [reply pg_config.js]: @python4eg  This config was changed.

**Pull request closed without merging**